src/licenseRequests.json

```diff
@@ -0,0 +1,35 @@
+    "fields": {
+      "fullname": "Historical Permission Notice and Disclaimer",
+      "shortIdentifier": "HPND",
+      "userEmail": "erick@mail.com",
+      "submissionDatetime": "2018-06-15 15:30:00",
+      "xml": "<?xml version='1.0' encoding='UTF-8'?> <SPDXLicenseCollection xmlns='http://www.spdx.org/license'> <license isOsiApproved='true' licenseId='HPND' name='Historical Permission Notice and Disclaimer'> <crossRefs> <crossRef>http://www.opensource.org/licenses/HPND</crossRef> </crossRefs> <notes> This license has been voluntarily deprecated by its author. Per the copy of this license at http://www.opensource.org/licenses/HPND Explanation: * Angle brackets hold 'fields', e.g. &lt;copyright holder&gt;. * Square brackets hold optional text, e.g. [or &lt;related entities&gt;]. * A license can have variations in capitalization and whitespace, and still be considered an instance of this template. It may be possible to construct a grammatically incorrect license from this template, or one that lacks a disclaimer, or one that includes a double-disclaimer. That is acceptable, as long as it remains impossible to construct a non-OSD-compliant license that matches the pattern. </notes> <text> <titleText> <p>Historical Permission Notice and Disclaimer</p> </titleText> <copyrightText> <p>&lt;copyright notice&gt;</p> </copyrightText> <p>Permission to use, copy, modify and distribute this software and its documentation for any purpose and without fee is hereby granted, provided that the above copyright notice appear in all copies<optional>,</optional> <optional>and</optional> that both <optional>that</optional> <optional>the</optional> copyright notice and this permission notice appear in supporting documentation<optional>, and that the name <optional>of</optional> <alt match='.*' name='copyrightHolder0'>&lt;copyright holder&gt;</alt> <alt match='.*' name='orRelated'>&lt;or related entities&gt;</alt> not be used in advertising or publicity pertaining to distribution of the software without specific, written prior permission</optional>. <optional> <alt match='.*' name='copyrightHolder1'>&lt;copyright holder&gt;</alt> makes no representations about the suitability of this software for any purpose. It is provided 'as is' without express or implied warranty.</optional> </p> <optional> <p> <alt match='.*' name='copyrightHolder2'>&lt;copyright holder&gt;</alt> DISCLAIMS ALL WARRANTIES WITH REGARD TO THIS SOFTWARE, INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS<alt match='[.,]?' name='period'>.</alt> IN NO EVENT SHALL <alt match='.*' name='copyrightHolder3'>&lt;copyright holder&gt;</alt> BE LIABLE FOR ANY SPECIAL, INDIRECT OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.</p> </optional> </text> </license> </SPDXLicenseCollection>"
+    }
+  }
+]
```

Comment:
Can you please put this file in the static folder? or Create another folder for testing purposes and add a README.md in that folder. If you require more files to be loaded for testing you can put all of them in that folder.if would be easier to distinguish and remove them after testing. Putting testing files in the main folder is not good.

Reply:
Great, I am going to create that folder. Thanks for the suggestion!

Reply:
Fine, I am going to create the folder and add both the readme file and the testing file.